day3-b.js

```diff
@@ -0,0 +1,33 @@
+//**Hello Ramin this is part 1 of day 3 :D **//
+//i am running this online to get the input here : https://www.codechef.com/ide //
+process.stdin.resume();
+process.stdin.setEncoding('utf8');
+
+
+process.stdin.on('data', function(data) { 
+    
+    var test = data.split("\n")
+		var zero = 0 ;
+	var one = 0 ;
+	var gamma= "";
+	var epsilon =""
+		let i = 0
+	 while ( i < test[1].length )  {
+		for(let j=0 ; j<test.length ; j++)	{		
+		if (test[j][i] === "0"){
+			zero += 1
+		} else one += 1 
+		}
```

Comment:
Definitely learn how to format your code well. This is very hard to read and a reviewer could have missed that the bracket was wrongly indented

```suggestion
		} else {
		  one += 1 
		}
              }
```